l3xz-fw_leg-controller.ino

```diff
@@ -62,12 +62,31 @@ static CanardPortID const ID_LED1                = 1005U;
 static SPISettings  const MCP2515x_SPI_SETTING{1000000, MSBFIRST, SPI_MODE0};
 static SPISettings  const AS504x_SPI_SETTING{1000000, MSBFIRST, SPI_MODE1};
 
+static const uavcan_node_GetInfo_Response_1_0 GET_INFO_DATA = {
+    /// uavcan.node.Version.1.0 protocol_version
+    {1, 0},
+    /// uavcan.node.Version.1.0 hardware_version
+    {1, 0},
+    /// uavcan.node.Version.1.0 software_version
+    {0, 1},
+    /// saturated uint64 software_vcs_revision_id
+    NULL,
```

Comment:
Well, I guess it would be nice to insert commit hash here but I'm not quite sure how to achieve that using `arduino-cli`. In any case it should be done automagically during build time, manual copy'n'paste is sure to lead to forgetting of updating the the revision id.